src/stylus/components/_timeline.styl

```diff
@@ -3,10 +3,24 @@
 
 v-timeline($material)
   &:before
-    background: $material.dividers
+    background: $material.timeline.active
+
+  .v-timeline-item
+    &__dot
+      background: $material.cards
+
+    &:nth-child(odd):not(.v-timeline-item--right), &--left
+      .v-timeline-item__body:before
+        border-color: transparent transparent transparent black
```

Comment:
Is black right for dark theme too? If it doesn't use the $material variable it shouldn't be in this section